src/App/Pages/Settings/ExportVaultPage.xaml.cs

```diff
@@ -65,5 +65,13 @@ private async void ExportVault_Clicked(object sender, EventArgs e)
                 await _vm.ExportVaultAsync();
             }
         }
+
+        void _fileFormatPicker_SelectedIndexChanged(object sender, EventArgs e)
```

Comment:
Method naming convention is irregular here, maybe use something like `FileFormat_Changed`